doc/developer-guide/developer-guide.rst

```diff
@@ -0,0 +1,122 @@
+.. _Developer_Guide:
+
+###################
+CBP Developer Guide
+###################
+
+The CBP software consists of a typical salobj CSC, a firmware written in DMC and a TCP/IP socket controller written in python.
+
+.. glossary::
```

Comment:
Please make all such anchors (I'm not sure what Sphinx calls them) globally unique, e.g. by prefixing with `ts.cbp.` (in this case `ts.cbp.glossary`).

That avoids ambiguity when linking documentation together.

(This may need to be fixed in your documentation template).

Reply:
Ping

Reply:
Changes made, but `.. glossary::` isn't an anchor.